test/index.js

```diff
@@ -0,0 +1,112 @@
+import test from 'tape';
+import compose from '../examples/compose';
+
+test('compose function', (assert) => {
+  const actual = typeof compose;
+  const expected = 'function';
+
+  assert.equal(actual, expected,
```

Comment:
using `assert.equal(actual, expected` tests tend to read more clearly.

Also, using equal combined with a clear description of what it is your testing, failed tests read like high-quality bug reports.